apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/ManifestItem/index.tsx

```diff
@@ -78,40 +78,44 @@ export default function ManifestItem({
     return manifest.icon?.trim();
   }, [manifest.icon]);
 
+  const clearSigningEnabled = useMemo(() => {
+    return manifest?.categories.includes("clear signing");
+  }, [manifest?.categories]);
+
   return (
     <TouchableOpacity disabled={isDisabled} onPress={handlePress}>
       <Flex flexDirection="row" alignItems="center" height={72} paddingX={4} paddingY={2}>
         <AppIcon isDisabled={isDisabled} size={48} name={manifest.name} icon={icon} />
         <Flex marginX={16} height="100%" flexGrow={1} flexShrink={1} justifyContent={"center"}>
-          <Flex flexDirection="row" alignItems={"center"} mb={2}>
+          <Flex flexDirection="row" alignItems={"center"} mb={2} columnGap={4}>
```

Comment:
If the fact that margins are used for everything else is the main argument, then okay

Otherwise, I think gap is a better practice since we're using flex, and it's cleaner for managing spacing in this context. As we deal with flexbox and want to add spacing between items, it's better to avoid affecting the first or last element